### ISSUE TITLE
Submitting form after reCaptcha validation throws error

### ISSUE THREAD

**Issue by medoingthings** [verbb/formie#1275]
### Describe the bug

When a user submits a valid form, but forgets to check the reCaptcha v2 checkbox, the form throws an error. When the user then checks the reCaptcha checkbox and submits the form, the following error is thrown:
`No submission exists with the ID "123456"`

This only happens when the reCaptcha checkbox is the only error. 

### Sidenote
My expectation was that there might be a separate error message displayed below the reCaptcha checkbox. At first I set the "Error Message Position" form setting to "none" so that only the field errors are shown. But when the reCaptcha failed validation (because it wasn’t clicked) there was no hint that the user needs to click the reCaptcha checkbox (even though it is pretty obvious, but still…).

### Steps to reproduce

1. Setup form with reCaptcha v2 (Checkbox) (I use the HubSpot integration in my form)
2. Fill out form, but don’t click the reCaptcha checkbox
3. Page reloads, now click the reCaptcha checkbox
4. Submit again

Error: `No submission exists with the ID "123456"`

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.6.1

### Plugin version

2.0.21

### Multi-site?

Yes

**Comment by engram-design** [comment]
Hmmm, so the error message should be shown directly underneath the checkbox if you fail to click it. It shouldn't reload the page either (even if you have client-side validation disabled, it will still evaluate this before submitting to the server).

<img width="433" alt="image" src="https://user-images.githubusercontent.com/1221575/216799325-f1d8f5b9-6e9f-4fe4-a2f1-b662e1c2c8e9.png">

The error message `No submission exists ...` only shows up when you're trying to submit a submission that's already been submitted.

So something is clearly going wrong here, and judging from your [other issue](https://github.com/verbb/formie/issues/1274), reCaptcha in general. Are there any JS errors in the web dev console? Are you using Vue or similar on your front-end? Anything else worth noting about your setup that might be considered different to normal?

Would it be possible to send through a support request from within Formie itself? Under Formie > Settings > Get Support, you can pick the form you need help with. That will send us your form configuration and settings (but none of your API keys, etc). Hopefully from that, I'll be able to replicate it on my end, as all captchas are working on our test installs!

**Comment by medoingthings** [comment]
This seems to be resolved with your fix from [my other issue](https://github.com/verbb/formie/issues/1274). 

Small detail: I used the theming options in order to use my own css classes, including the field error class. The front-end validation for the reCaptcha error still uses the class `fui-error-message`, so it doesn’t respect the theming options. But it’s kind of an edge case and I can easily live with it. Just wanted to mention it in case that’s not intentional.

**Comment by engram-design** [comment]
Good call, that's pretty much the only instance left where we don't refer to theme config classes.

Updated for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.22](https://github.com/verbb/formie/releases/tag/2.0.22)
